src/modules/i18n/reducers/index.js

```diff
@@ -1,6 +1,15 @@
 // @flow
 
-import { handleAction } from 'redux-actions'
-import { setUiDirectionAction } from '../actions/setUIDirection'
+import { type ActionType, handleAction, type ReduxReducer } from 'redux-actions'
+import setUIDirection from '../actions/setUIDirection'
 
-export default handleAction(setUiDirectionAction, (state, action) => action.payload, 'ltr')
+export type DirectionStateType = 'ltr' | 'rtl'
+
+const uiDirectionReducer: ReduxReducer<DirectionStateType, { type: 'SET_UI_DIRECTION', payload: 'ltr' | 'rtl' }> =
```

Comment:
You could just use `SetUiDirectionActionType` instead of `{ type: 'SET_UI_DIRECTION', payload: 'ltr' | 'rtl' }`